sparse_strips/vello_cpu/tests/issues.rs

```diff
@@ -0,0 +1,285 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Tests for Github issues.
+
+use crate::util::{check_ref, get_ctx, render_pixmap};
+use vello_common::color::palette::css::{DARK_BLUE, LIME};
+use vello_common::kurbo::{BezPath, Stroke};
+use vello_common::peniko::Fill;
+
+mod util;
+
+// All issue numbers starting from here refer to the issue numbers in the old repo
+// at https://github.com/LaurenzV/cpu-sparse-experiments/issues.
+
+#[test]
```

Comment:
We should have a link to the relevant issue, i.e.
```suggestion
#[test]
// https://github.com/LaurenzV/cpu-sparse-experiments/issues/2
```

Reply:
Will just do this for all then!